### DISCUSSION TITLE
Patch requests moved to ReVanced Patches repository

### DISCUSSION THREAD

**Discussion by Sculas** [ReVanced/revanced-discussions#589] [Announcements]
## 🧩 Patches

From now on issues for patch suggestions and requests should be created in the https://github.com/revanced/revanced-patches repository instead of the ReVanced Discussions.

Start requesting patches [here](https://github.com/ReVanced/revanced-patches/issues/new?template=feature_request.yml).

**Comment by anil-al** [comment]
But those were suggested in Revanced Suggestions will still be valid, won't they?

**Reply by oSumAtrIX** [reply]
Yes

**Comment by thisisshihan** [comment]
Remini - AI Photo Enhancer patcher please?
